index.html

```diff
@@ -2467,22 +2467,22 @@ <h5>
         <ol>
           <li id="id148">
             <p its-locale-filter-list="en" lang="en">The base text contains one or more Han characters. Rules for separating terms can be found in <cite>Basic Rules of Hanyu Pinyin Orthography</cite> (GB/T 16159—2012).</p>
-            <p its-locale-filter-list="zh-hans" lang="zh-hans" class="checkme">基文为一个或多个汉字构成的词语。分词连写规则可依据《汉语拼音正词法基本规则》（GB/T 16159—2012）。</p>
+            <p its-locale-filter-list="zh-hans" lang="zh-hans">基文为一个或多个汉字构成的词语。分词连写规则可依据《汉语拼音正词法基本规则》（GB/T 16159—2012）。</p>
             <p its-locale-filter-list="zh-hant" lang="zh-hant">基文为一个或多个汉字构成的词语。分词连写规则可依据《汉语拼音正词法基本规则》（GB/T 16159—2012）。</p>
           </li>
           <li id="id149">
             <p its-locale-filter-list="en" lang="en">Annotations sometimes appear below the Han characters.</p>
-            <p its-locale-filter-list="zh-hans" lang="zh-hans" class="checkme">注文有时在下方。</p>
+            <p its-locale-filter-list="zh-hans" lang="zh-hans">注文有时在下方。</p>
             <p its-locale-filter-list="zh-hant" lang="zh-hant">注文有时在下方。</p>
           </li>
           <li id="id150">
-            <p its-locale-filter-list="en" lang="en">Both the phonetic annotations and the base text are separated at word boundaries. The adjacent annotations are separated by a space approximately 1/2 em wide, while the tracking inside the base text is usually normal.</p>
-            <p its-locale-filter-list="zh-hans" lang="zh-hans" class="checkme">正文与标音双方皆分词连写。相邻基文之间有约1/2em的空格隔开，基文内部字距通常正常。</p>
+            <p its-locale-filter-list="en" lang="en">Both the phonetic annotations and the base text are separated at word boundaries. The adjacent annotations are separated by an approximately 1/2 em wide space, while the tracking inside the base text is usually normal.</p>
+            <p its-locale-filter-list="zh-hans" lang="zh-hans">正文与标音双方皆分词连写。相邻基文之间有约1/2em的空格隔开，基文内部字距通常正常。</p>
             <p its-locale-filter-list="zh-hant" lang="zh-hant">正文与标音双方皆分词连写。相邻基文之间有约1/2em的空格隔开，基文内部字距通常正常。</p>
           </li>
           <li id="id151">
-            <p its-locale-filter-list="en" lang="en">Many word-based annotations indicate the logic of the whole sentence, rather than merely the pronunciation: these phonetic annotations have sentence case, as well as punctuation marks which follow the previous annotations.</p>
-            <p its-locale-filter-list="zh-hans" lang="zh-hans" class="checkme">许多分词连写标音用例体现出对整个句子标音的逻辑，而非简单对词语标音：注文有句首大写、专名首字母大写。有标点，标点跟随左侧（前方）注文。</p>
+            <p its-locale-filter-list="en" lang="en">Many word-based annotations indicate the logic of the whole sentence, rather than merely the pronunciation: these phonetic annotations have capitalized sentences and capitalized proper names. Punctuations follow the previous annotation text.</p>
```

Comment:
I'm not sure what "Punctuations follow the previous annotation text." means.  Note btw, that it should read "Punctuation follows...".

Reply:
It means the punctuation immediately follows the ruby text (left in horizontal writing mode and above in vertical writing mode).

For example, it should be:

<img width="545" alt="1" src="https://user-images.githubusercontent.com/2863444/75141016-0a863800-572b-11ea-9d36-1cfddfc432d4.png">

instead of something like the following:

<img width="545" alt="2" src="https://user-images.githubusercontent.com/2863444/75141037-14a83680-572b-11ea-9a83-a1cf19419842.png">

Reply:
That's much clearer now. I strongly recommend including the top image as a figure, and saying something like "Punctuation may also be included in these annotations, but is kept with a preceding annotation, as shown in Fig XX, and doesn't appear over the punctuation in the base text."

Reply:
I just pushed a commit to clarify this point.